Trippy/View Controllers/SavedCollectionsViewController.m

```diff
@@ -7,25 +7,35 @@
 
 #import "SceneDelegate.h"
 #import "SavedCollectionsViewController.h"
+#import "CreateCollectionViewController.h"
+#import "ListTableViewController.h"
 #import "LoginViewController.h"
 #import "LogoutHandler.h"
 
-@interface SavedCollectionsViewController () <LogoutHandlerDelegate>
-@property (strong, nonatomic) LogoutHandler *handler;
+@interface SavedCollectionsViewController () <LogoutHandlerDelegate, CreateCollectionDelegate>
+@property (strong, nonatomic) LogoutHandler *logoutHandler;
 @end
 
 @implementation SavedCollectionsViewController
 
 - (void)viewDidLoad {
+    self.listType = kCollection;
+    
     [super viewDidLoad];
     
     // Set up Parse interface
-    self.handler = [[LogoutHandler alloc] init];
-    self.handler.delegate = self;
+    self.logoutHandler = [[LogoutHandler alloc] init];
```

Comment:
Similar comment as with the other delegate, to consider making an `initWithDelegate:` instead.